recipe/meta.yaml

```diff
@@ -2,10 +2,11 @@
 # if build_num is reset to 0 (for new version), update increment for blas_minor below
 {% set build_num = 8 %}
 {% set version_major = version.split(".")[0] %}
-{% set blas_major = "2" %}
+{% set blas_major = "3" %}
```

Comment:
not sure if this is really necessary; it's a rather small change in the conda-forge blas infrastructure, but a change nevertheless. But it's perhaps not a bad idea to start with a clean slate - there were some collisions on this between 3.8.0 & 3.9.0 until `blas_minor` got introduced recently.

Reply:
> But it's perhaps not a bad idea to start with a clean slate 

That's not a good enough reason to rebuild all downstream packages. Changing the major version will require a rebuild of all downstream packages.

Reply:
Wasn't aware that this requires a rebuild, agreed that that's not worth it.